lib/google_search_data_viewer/keyword/google_search_parser.ex

```diff
@@ -0,0 +1,61 @@
+defmodule GoogleSearchDataViewer.Keyword.GoogleSearchParser do
+  @css_search_selectors %{
+    top_adwords: "#tads > .uEierd a.sVXRqc[href]",
+    top_non_adwords: ".MhgNwc a[href]",
+    non_adwords: ".yuRUbf a[href]",
+    bottom_adwords: "#bottomads .uEierd a.sVXRqc[href]"
+  }
+
+  def get_url_data_from_html(html) do
```

Comment:
The existing name is quite generic, you can think about a new name for it. For example:

```suggestion
  def get_urls_statistic(html) do
```